src/main/kotlin/io/github/anki/anki/service/DeckService.kt

```diff
@@ -78,6 +87,17 @@ class DeckService(
         }
     }
 
+    private fun sendEventToKafka(deckId: String, deckEventType: DeckEventType) {
```

Comment:
try to realise function like
private fun sendEventToKafka(deckId: String, deckEventType: DeckEventType): Mono`<Unit`> {
...
return kafkaProducer.send(KafkaTopic.Deck.EVENT, deckEvent).doOnNext...
}